templating.rst

```diff
@@ -301,6 +300,23 @@ When working with template inheritance, here are some tips to keep in mind:
           {{ parent() }}
       {% endblock %}
 
+.. caution::
+
+    When using ``extends``, a child template is forbidden to define template
+    parts outside of a block. The following code throws a ``SyntaxError``:
+
+    .. code-block:: html+twig
+
+        {# app/Resources/views/blog/index.html.twig #}
+        {% extends 'base.html.twig' %}
+
+        {# the line below is not captured by a "block" tag #}
+        <div class="alert">Some Alert</div>
```

Comment:
```suggestion
        My cool blog posts
```

to make it more clear?

Reply:
The intent is to have an "all page" valid except this line. Not to duplicate the example with valid/invalid states. That is something that ppl can do by mistake the first times they use Twig, I've seen that in a lot of trainings.